src/Screens/Explore/DiveSite.tsx

```diff
@@ -152,7 +152,7 @@ const DiveSite: FunctionComponent<DiveSiteProps> = ({ navigation, route }) => {
 
     // just make call to fetch beach if no images, since we're only calling this hook once now
     dispatch(handleFetchDiveSite(currentSpotId));
-  }, [currentSpotId]); // eslint-disable-line react-hooks/exhaustive-deps
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
why did you change this?

Reply:
Oh, that was a mistake. Was trying to fix a very strange bug infinity loop bug so I was fiddling with the dependenncy list to try to understand where it came from. Reinstalled now.